docs/deployment/linux/red-hat.txt

```diff
@@ -0,0 +1,141 @@
+.. _red-hat:
+
+============================
+Run CrateDB on Red Hat Linux
+============================
+
+CrateDB maintains the official RPM repositories for:
+
+- `Red Hat Linux 7`_
+- `Red Hat Linux 6`_
+
+.. _Red Hat Linux 7: https://www.redhat.com/en/resources/whats-new-red-hat-enterprise-linux-7
+.. _Red Hat Linux 6: https://www.redhat.com/en/resources/red-hat-enterprise-linux-6
+
+Both of these work with RedHat Enterprise Linux, CentOS, and Scientific Linux.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Configure YUM
+=============
+
+All CrateDB packages are signed with GPG.
+
+To get started, you must import the CrateDB public key, like so:
+
+.. code-block:: sh
+
+   sh$ sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
+
+You must then install the CrateDB repository definition.
+
+For Red Hat Linux 7, run:
+
+.. code-block:: sh
+
+   sh$ sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/noarch/crate-release-7.0-1.noarch.rpm
+
+For Red Hat Linux 6, run:
+
+.. code-block:: sh
+
+   sh$ sudo rpm -Uvh https://cdn.crate.io/downloads/yum/6/x86_64/crate-release-6.5-1.noarch.rpm
+
+The above command creates two new repository configuration files:
+
+- ``/etc/yum.repos.d/crate-stable.repo``
+- ``/etc/yum.repos.d/crate-testing.repo``
+
+CrateDB provides a stable and a testing release channel. At this point, you
+should select which one you wish to use.
+
+By default, `YUM`_ (Red Hat's package manager) will use the stable repository.
+This is because the testing repository's configuration marks it as disabled.
+
+.. _YUM: https://access.redhat.com/solutions/9934
+
+If you would like to enable to testing repository, open the
+``crate-testing.repo`` file and set this:
+
+.. code-block:: text
+
+   enabled=1
+
+You can now install CrateDB.
+
+Install CrateDB
+===============
+
+With everything set up, you can install CrateDB, like so:
+
+.. code-block:: sh
+
+   yum install crate
+
+CrateDB is now installed, but not running.
+
+Running and Controlling CrateDB
+===============================
+
+With Red Hat Linux 7, you can control the ``crate`` service like so:
+
+.. code-block:: sh
+
+   sh$ sudo systemctl COMMAND crate
+
+With Red Hat Linux 6, you should use:
+
+.. code-block:: sh
+
+   sh$ sudo service crate COMMAND
+
+In both instances, replace ``COMMAND`` with ``start``, ``stop``, ``restart``,
```

Comment:
optional: with the command `status`  you can check if the service is active and running